recipes/scikit-beam/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set version = "0.0.8" %}
+
+package:
+    name: scikit-beam
+    version: {{ version }}
+
+source:
+    url: https://github.com/scikit-beam/scikit-beam/archive/v{{ version }}.tar.gz
+    fn: scikit-beam-v{{ version }}.tar.gz
+    sha256: 084690482e418defbe36bb0969133d8de22d21b4a06e09c62ccb6b1717d2e2b4
+
+build:
+  number: 0
+  script:
+    # FIXME: This is a hack to make sure the environment is activated.
+    # The reason this is required is due to the conda-build issue
+    # mentioned below.
+    #
+    # https://github.com/conda/conda-build/issues/910
+    #
+    - source activate "${CONDA_DEFAULT_ENV}"  # [unix]
+    - python setup.py build_ext -i
+    - python setup.py install --single-version-externally-managed --record=record.txt
+  # v0.0.8 of scikit-beam is not windows compatible because the setup.py is not formatted correctly
+  # to build the extensions that have some linux-specific include directives. Once there is a 0.0.9 tag,
+  # we can remove thsi next line and be good to go
+  skip: True  # [win]
+
+requirements:
+    build:
+        - toolchain
+        - python
+        - setuptools
+        - numpy x.x
+        - six
+        - cython
+        - autoconf    # [osx]
+    run:
+        - python
+        - numpy x.x
+        - scipy
+        - six
+        - xraylib    # [not win]
+        - scikit-image
+        - lmfit 0.8.3
+
+test:
+    imports:
+        - skbeam
+        - skbeam.io
+        - skbeam.core
+        - skbeam.core.accumulators
+        - skbeam.core.constants
+        - skbeam.core.fitting
+        - skbeam.core.fitting.base
+        - skbeam.ext
+        # xraylib is not conda installable on windows yet  # [not win]
+        - skbeam.core.fitting    # [not win]
+        - skbeam.core.constants.xrf    # [not win]
+        # has linux native library imports  # [not win]
+        - skbeam.ext.ctrans    # [not win]
```

Comment:
I recommend you to add:

```yaml
    commands:
        - conda inspect linkages -n _test scikit-beam  # [not win]
        - conda inspect objects -n _test scikit-beam  # [osx]
```

in all packages that are linking to something. That helps to debug issues.